### PR TITLE
CDK GH Action

### PR DESCRIPTION
**Pull request by Forsarion**
This is duplicate of - https://github.com/AlexeyDemedetskiy/mywarmind.net/pull/4

### CODE REVIEW

**Comment by Forsarion** [comment]
@AlexeyDemedetskiy RFR

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+![mywarmind.net CI](https://github.com/Forsarion/mywarmind.net/workflows/mywarmind.net%20CI/badge.svg)
```
Comment (AlexeyDemedetskiy) [inline README.md]: Should be still your repo?

Comment (Forsarion) [reply README.md]: No, should be yours.

Comment (Forsarion) [reply README.md]: Updated.

**Pull request merged**